apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -185,7 +185,7 @@ get_top_empty_chain(_Config) ->
     ok.
 
 get_top_non_empty_chain(_Config) ->
-    ok = aecore_suite_utils:mine_one_block(aecore_suite_utils:node_name(?NODE)),
+    {ok, _} = aecore_suite_utils:mine_blocks(aecore_suite_utils:node_name(?NODE), 1),
```

Comment:
I see match of return value (without binding of variables) here though not in most other calls to `aecore_suite_utils:mine_blocks`: maybe slipped in other points? Anyway matching seems not relevant because of exception `erlang:error` raised in case of not-ok.

Reply:
Yes, the patten matching is not neccessary